src/System.Data.Common/src/System/Data/DataReaderExtensions.cs

```diff
@@ -0,0 +1,182 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Data.Common;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Data
+{
+    public static partial class DataReaderExtensions
```

Comment:
Sorry about all the back and forth on this, but the `partial` no longer seems necessary here.